projects/vdk-control-cli/setup.cfg

```diff
@@ -54,6 +54,7 @@ install_requires =
     tabulate
     requests_oauthlib>=1.0
     urllib3>=1.26.5
+    croniter==1.3.4
```

Comment:
Do we need to pin the version ?

Reply:
Yes, It's a good practice to pin versions in Command-Line interface - because 
A) it's more secure as you mention 
b) it's more stable - CLI won't break if dependency library releases a new version with breaking change. 

It's less of a good practice in a library because it restricts what the users of the library can do - users would want to install their own dependencies and likely will use newer version and will conflict with pinned one.

While vdk-control-cli is a library it's often packaged alongside vdk-core and plgins (which are libraries) and is used in same environment so I prefer to leave them unpinned for now.